docs/samples/Microsoft.ML.Samples/Dynamic/Transforms/ReplaceMissingValues.cs

```diff
@@ -13,27 +13,18 @@ public static void Example()
             // Create a new ML context, for ML.NET operations. It can be used for exception tracking and logging, 
             // as well as the source of randomness.
             var mlContext = new MLContext();
+
+            // Get a small dataset as an IEnumerable and convert it to an IDataView.
             var samples = new List<DataPoint>()
             {
-                new DataPoint(){ Label = 3, Features = new float[3] {1, 1, 0} },
-                new DataPoint(){ Label = 32, Features = new float[3] {0, float.NaN, 1} },
-                new DataPoint(){ Label = 5, Features = new float[3] {-1, 2, -3} },
-                 new DataPoint(){ Label = 9, Features = new float[3] {-1, 6, -3} },
+                new DataPoint(){ Features = new float[3] {float.PositiveInfinity, 1, 0} },
+                new DataPoint(){ Features = new float[3] {0, float.NaN, 1} },
+                new DataPoint(){ Features = new float[3] {-1, 2, -3} },
+                new DataPoint(){ Features = new float[3] {-1, float.NaN, -3} },
             };
-            // Convert training data to IDataView, the general data type used in ML.NET.
             var data = mlContext.Data.LoadFromEnumerable(samples);
 
-            // ReplaceMissingValues is used to create a column where missing values are replaced according to the ReplacementMode.
-            var meanPipeline = mlContext.Transforms.ReplaceMissingValues("MissingReplaced", "Features", MissingValueReplacingEstimator.ReplacementMode.Mean);
-
-            // Now we can transform the data and look at the output to confirm the behavior of the estimator.
-            // This operation doesn't actually evaluate data until we read the data below.
-            var meanTransformer = meanPipeline.Fit(data);
-            var meanTransformedData = meanTransformer.Transform(data);
-
-            // We can extract the newly created column as an IEnumerable of SampleDataTransformed, the class we define below.
-            var meanRowEnumerable = mlContext.Data.CreateEnumerable<SampleDataTransformed>(meanTransformedData, reuseRowObject: false);
-
+            // Default ReplaceMode:
             // ReplaceMissingValues is used to create a column where missing values are replaced according to the ReplacementMode.
```

Comment:
>// ReplaceMissingValues  [](start = 12, length = 24)

// Here we use the default replacement mode, which replaces the value with the default value for its `type`.